src/main/java/roomescape/dao/ReservationTimeDAO.java

```diff
@@ -0,0 +1,48 @@
+package roomescape.dao;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+import javax.sql.DataSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import roomescape.dto.TimeCreateRequestDto;
+import roomescape.model.ReservationTime;
+
+@Repository
+public class ReservationTimeDAO {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public ReservationTimeDAO(DataSource dataSource) {
+        this.jdbcTemplate = new JdbcTemplate(dataSource);
+    }
+
+    public List<ReservationTime> findAllTimes() {
+        return jdbcTemplate.query("SELECT * FROM reservation_time", rowMapper);
+    }
```

Comment:
클래스 이름에서 이미 도메인을 노출하고 있어서 생략해도 괜찮지 않을까요?

```suggestion
    public List<ReservationTime> findAll() {
        return jdbcTemplate.query("SELECT * FROM reservation_time", rowMapper);
    }
```

Reply:
insert 와 delete 는 생략했는데 findAll 만 붙이고 있었네요 😅
감사합니다!